TeaDiary.dataaccess/Migrations/Configuration.cs

```diff
@@ -61,10 +69,14 @@ protected override void Seed(TeaDiaryContext context)
                     UpdateDate = new DateTime(2019,1,1)
                 }
             };
-            
-            
-            context.Users.AddOrUpdate(initialUser);
-            context.Teas.AddOrUpdate(initialTeas);
+
+            foreach (Tea initialTea in initialTeas)
+            {
+                context.Teas.AddOrUpdate(initialTea);
+            }
```

Comment:
Why don't you just pass the array as you did before?

Reply:
Well, I was experimenting with .Add which couldn't take array as param and forgot to change it back

Reply:
Dew it